modules/core/src/v2/coins/cspr.ts

```diff
@@ -0,0 +1,83 @@
+import { BaseCoin, KeyPair, SignedTransaction, VerifyAddressOptions, VerifyTransactionOptions } from '../baseCoin';
+import { NodeCallback } from '../types';
+import { BitGo } from '../../bitgo';
+import { Cspr as CsprAccountLib } from '@bitgo/account-lib';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+import Bluebird = require('bluebird');
+
+export class Cspr extends BaseCoin {
+    
+    protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+
+    constructor(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>) {
+      super(bitgo);
```

Comment:
It's a bit odd to alternate between 4 and 2 space indent. Not a blocker though. You can add the following comment at the top of the file to have prettier auto-format any changes so you don't need to think about this:
```
/*
 * @prettier
 */
```